app/routers/post.py

```diff
@@ -43,8 +35,64 @@ def get_posts(
         .offset(skip)
         .all()
     )
+
+    post_comments = {}
+    for post in posts:
+        post_id = post[0].id
+        comments = (
+            db.query(models.Comment).filter(models.Comment.post_id == post_id).all()
+        )
+        post_comments[post_id] = comments
+
+    post_output = []
+    for post in posts:
+        post_output.append(
```

Comment:
If we go down this path, I suggest creating Pydentic model for this format of response and use it instead of plain dict